apps/etterlatte-behandling/src/main/resources/db/migration/V113__opprettet_tid_sak.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE sak ADD COLUMN opprettet TIMESTAMP;
+
+UPDATE sak as s
+SET opprettet = (select behandling_opprettet from behandling as b where b.sak_id = s.id);
+
+UPDATE sak
+SET opprettet = now()::timestamp
+WHERE sak.opprettet IS NULL;
```

Comment:
Hmm, synes kanskje ikke dette er helt heldig. I mitt hode er det bedre å _ikke_ ha dato for opprettelse enn å ha en feilaktig og misvisende dato.

Reply:
finnes det kanskje noe som kan brukes som kilde til faktisk opprettet dato? noe i grunnlag?

Reply:
Hmm ok ja kan fjerne denne og heller ha null på de vi ikke vet.

Det viktigste er vel å populere nye fremover

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/4604/commits/f712eba4d2f20b3d1214448f6a665e99d654b39b